README.md

```diff
@@ -47,12 +47,12 @@ Python API:
 >>> duration.get_days_and_seconds()
 (365.0, 10800.0)
 >>> date_time + duration
-2001-01-01T03:00:00Z
+<metomi.isodatetime.data.TimePoint: 2001-01-01T03:00:00Z>
```

Comment:
Is this new `__repr__` format too verbose?

Reply:
I don't think it matters too much.

The original intension of `repr` was to be **repr**oducible but these days it's a bit of a free-for-all (I've sometimes miss-use them on-purpose), whatever works.

Other code shouldn't really be relying on the `repr` format so we should be free to change.